packages/cozy-harvest-lib/src/components/KonnectorModal.jsx

```diff
@@ -68,8 +53,6 @@ export class KonnectorModal extends PureComponent {
    * Next tasks: remove these methods and rewrite the override
    */
   componentDidMount() {
-    const { client } = this.context
-    Account.copyWithClient(client)
```

Comment:
Where did that go ? Was it not important ? Also, I am a bit surprised that the result of `Account.copyWithClient(client)` is not used, I thought copyWithClient was supposed to return a copy that had the client in it.

Reply:
Yes `copyWithClient` returns a new copy with a client in it, and it wasn't used here. The call to copyWithClient was not useful here, none of the functions of `Account` use the cozy-client.